scripts/reconcile-antigravity-review-state.py

```
@@ -0,0 +1,300 @@
+        if value.get(preserved) != baseline.get(preserved):
+            raise ReconcileError(f"proposed reviewed evidence changed human-owned {preserved}")
+    installer = value.get("installer")
+    binary = value.get("installed_binary")
+    if not isinstance(installer, dict) or not isinstance(binary, dict):
+        raise ReconcileError("proposed reviewed evidence is missing installer/binary metadata")
+    if installer.get("official_url") != OFFICIAL_URL or installer.get("final_url") != OFFICIAL_URL:
+        raise ReconcileError("proposed reviewed evidence changed the fixed installer origin")
+    installer_sha = sha256(installer.get("sha256"), "proposed installer SHA-256")
+    payload_sha = sha256(binary.get("sha256"), "proposed binary SHA-256")
+    if binary.get("relative_path") != EXPECTED_BINARY:
+        raise ReconcileError("proposed reviewed evidence changed the expected binary path")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)